configs/default/NERC-NEUTRONRCH7BT.config

```diff
@@ -3,9 +3,10 @@
 #define USE_ACC
 #define USE_GYRO
 #define USE_ACCGYRO_BMI270
-#define USE_FLASH
```

Comment:
This FC doesnt have flash memory for BB??

Reply:
guess it does.  it is difficult find official NeutronRC sources. 
https://newbeedrone.com/products/neutronrc-h7bt

Reply:
Board uses SD NAND and SDCard interface like Speedybee F7 V3.
The W25M02G is unsupported and not recommended.